components/app.js

```diff
@@ -12,11 +13,30 @@ class App {
     var newAverage = total / grades.length;
     this.pageHeader.updateAverage(newAverage)
   }
-  constructor(gradeTable, pageHeader) {
+  createGrade(name, course, grade) {
+    console.log(name, course, grade)
+    $.ajax({
+      headers: {
+        "X-Access-Token": "QL4ulI2r"
+      },
+      url: 'https://sgt.lfzprototypes.com/api/grades',
+      method: 'POST',
+      data: {
+        name: name,
+        course: course,
+        grade: grade
+      },
+      error: this.handleGetGradesError,
+      success: this.handleGetGradesSuccess
```

Comment:
There should be methods specifically for handling `createGrades` success and error responses.